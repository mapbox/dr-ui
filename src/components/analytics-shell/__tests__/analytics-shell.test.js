import renderer from 'react-test-renderer';
import { testCases } from './analytics-shell-test-cases.js';

describe('AnalyticsShell', () => {
  describe(testCases.basic.description, () => {
    let testCase;
    let wrapper;
    let tree;

    beforeEach(() => {
      testCase = testCases.basic;
      wrapper = renderer.create(testCase.element);
      tree = wrapper.toJSON();
    });

    test('renders as expected', () => {
      expect(tree).toMatchSnapshot();
    });
  });
});
