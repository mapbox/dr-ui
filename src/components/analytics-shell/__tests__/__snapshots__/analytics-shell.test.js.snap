// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AnalyticsShell Basic renders as expected 1`] = `"Hello world!"`;
